src/wirecloud/commons/static/js/wirecloud/ui/ResizeHandle.js

```diff
@@ -145,6 +145,7 @@
             dragboardCover.style.top = "0";
             dragboardCover.style.left = "0";
             dragboardCover.style.width = "100%";
+            dragboardCover.style.cursor = handleElement.dataset.cursor;
```

Comment:
To keep the cursor when resizing, I need a way to pass the cursor type to the cover. This solution might not be the cleanest but I found that better than passing it through 3 levels